web/src/routing/routes/UsersAdministration/Groups/GroupsTable/GroupsTable.tsx

```diff
@@ -0,0 +1,75 @@
+import { FC } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { Theme } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+import { Table } from "@components/Table/Table";
+
+import { Group } from "@janush-types/group";
+
+const columns = [
+  { name: "name", label: "Name" },
+  { name: "description", label: "Description" },
+  { name: "members", label: "Members" },
+  { name: "lastModified", label: "Last Modified" },
+];
+
+interface Props {
+  data: Group[];
+}
+
+const GroupsTable: FC<Props> = ({ data }) => {
+  const { TableHead, TableRow, TableCell, TableBody } = Table;
```

Comment:
That operation happens every re-render, I think we can move it outside of component.